claim/schema.py

```diff
@@ -114,6 +115,13 @@ def resolve_claims(self, info, **kwargs):
         if services:
             query = query.filter(services__service__code__in=services)
 
+        attachment_status = kwargs.get("attachment_status", None)
+
+        if attachment_status == 1:
+            query = query.filter(attachments__isnull=False)
+        elif attachment_status == 2:
+            query = query.filter(attachments__isnull=True)
+
```

Comment:
can you use static enums instead of literals for this check? It'll improve readability.